### PR TITLE
Group Notify Me

### LINKED ISSUE #142: Add "Notify Me" button
- When the “Notify Me” button is pressed, the user should be prompted to log in if they are not already.
- The user should be prompted to select the campus they would like notifications for.
- They should then be added to the Group inside of Rock that represented the Hub for the Selected Campus.
- They should see a physical prompt saying that they have been added to the group and can expect notifications to be send via their preferred log in method (if phone number was used, say phone. If email was used, say email).

### PR DESCRIPTION
**Pull request by rkdavidson**
# About
Closes #142 

Stubs out the flow for Group Notify me. Two major pieces missing that were worked around:
the logic for _when_ to show the button, and the actual mutation to invoke with values `userId`, `campusId`, and `groupPreferenceId`.

Flow should work otherwise.

# Testing
Go to any `CommunitySingle` page, like `/community/crew`.
Open the **Notify Me** button, and verify you are prompted to log in if you aren't already.
Verify your users' campus is preselected in the Campus select.
Verify you see the values console logged on submit.

# Screenshots
|Caption|Screenshot/GIF|
|---|---|
|Full Flow|CleanShot_2021-05-13_11 52 39|
|Desktop|CleanShot_2021-05-13_11 53 39|
|Mobile|CleanShot_2021-05-13_11 54 13|
|Step 1|CleanShot_2021-05-13_11 53 44|
|Step 2|CleanShot_2021-05-13_11 53 49|

### CODE REVIEW

File: components/CommunitySingle/CommunitySingle.js [hunk 1]
```diff
@@ -97,7 +115,36 @@ function CommunitySingle(props = {}) {
...
+            <Box as="p" color="subdued">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin ut
+              imperdiet erat, sed sodales lorem.
+            </Box>
+          </Box>
+          <Box
+            display="flex"
+            flex={1}
+            flexDirection="row"
+            justifyContent="center"
```
Comment (asleepingpanda) [inline components/CommunitySingle/CommunitySingle.js:133]: Could we try `justifyContent={{ _: 'center', lg: 'flex-end' }}`

**Review by asleepingpanda — DISMISSED** [review]
Looks great! Love the way it all came together. Functionality is perfect. Just one teensie-tiny UI update. Otherwise I'm good with this!

File: components/Modals/GroupNotifyMeModal/NotifyMeForm.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  const { currentUser } = useCurrentUser();
+  const [campusId, setCampusId] = useState(props.initialCampusId || '');
+  const modalDispatch = useModalDispatch();
+
+  const handleCampusSelect = event => {
+    setCampusId(event.target.value);
+  };
+
+  const handleSubmit = () => {
+    console.group('%cNotify Me form submitted', 'color: limegreen');
```
Comment (jordanwade) [inline components/Modals/GroupNotifyMeModal/NotifyMeForm.js:43]: Probably could kill these console.logs now

Comment (rkdavidson) [reply components/Modals/GroupNotifyMeModal/NotifyMeForm.js:43]: I can remove them, but this is the "handoff" point so purposely left them in for testing validation.
I'll leave them in, but commented out, until the final code is in place.

File: components/CommunitySingle/CommunitySingle.js (hunk 1 above)
Comment (rkdavidson) [reply components/CommunitySingle/CommunitySingle.js]: Updated in 5e40fa6

File: components/Modals/GroupNotifyMeModal/NotifyMeForm.js (hunk 2 above)
Comment (rkdavidson) [reply components/Modals/GroupNotifyMeModal/NotifyMeForm.js]: Updated in 5e40fa6

Comment (jordanwade) [reply components/Modals/GroupNotifyMeModal/NotifyMeForm.js]: O, okay 👍 Either way is good for me.

**Review by jordanwade — DISMISSED** [review]

**Review by jordanwade — APPROVED** [review]

**Pull request merged**